src/cmd/sources.rs

```diff
@@ -16,46 +16,46 @@ use crate::src::SourceGroup;
 use crate::target::{TargetSet, TargetSpec};
 
 /// Assemble the `sources` subcommand.
-pub fn new<'a, 'b>() -> App<'a, 'b> {
-    SubCommand::with_name("sources")
+pub fn new<'a>() -> Command<'a> {
+    Command::new("sources")
         .about("Emit the source file manifest for the package")
         .arg(
-            Arg::with_name("target")
-                .short("t")
+            Arg::new("target")
+                .short('t')
                 .long("target")
                 .help("Filter sources by target")
                 .takes_value(true)
-                .multiple(true)
+                .multiple_occurrences(true)
```

Comment:
What about `multiple_values(true)`?

Also below for the `package` and the `exclude` argument.